test/ruby_lsp_rails/hover_test.rb

```diff
@@ -146,55 +100,14 @@ class CompositePrimaryKey < ApplicationRecord
 
           **product_id**: integer (PK)
 
-          **note**: string
+          **note**: text
 
           **created_at**: datetime
 
           **updated_at**: datetime
         CONTENT
       end
 
-      test "handles `db/structure.sql` instead of `db/schema.rb`" do
```

Comment:
With the new approach, these two tests would be awkward, and I don't think really provide much value, so I'm proposing we delete them. (If we did want to keep them, we'd should could them to `server_test.rb` and adapt them).

Reply:
Yeah I think `server_test.rb` will be a better place for these tests. And since we do test against `schema_dump_path` being unsupported there, I think it's worth keeping these 2 tests there instead of deleting them.